src/github.com/getlantern/flashlight/app/settings.go

```diff
@@ -145,6 +145,14 @@ func (s *Settings) checkBool(data map[string]interface{}, name string, f func(bo
 	}
 }
 
+func (s *Settings) checkInt(data map[string]interface{}, name string, f func(int)) {
+	if v, ok := data[name].(int); ok {
```

Comment:
In the server:
* The code deals with numbers as 64 bits integers.
* Redis handles everything internally as strings, but the numbers are always parsed when crossing the Lua boundary so they are properly typed.